server/translations.ts

```diff
@@ -141,22 +142,45 @@ function getDocument(filePath) {
     };
   }
 
+  function getCommitBehindFromLatest(filename, commitHash) {
+    const commitHashes = execSync(
+      `cd ${CONTENT_ROOT} && git log --pretty=format:%H -- ${filename}`
+    )
```

Comment:
just use the `cwd` option?

```suggestion
    const commitHashes = execSync(`git log --pretty=format:%H -- ${filename}`, {
      cwd: CONTENT_ROOT,
    })
```

Reply:
Thank you :)

PR: https://github.com/mdn/yari/pull/8338/commits/6c9af9b159a28b1159736f748f787c28558e4fbd